src/test/java/redis/clients/jedis/TupleSortedSetTest.java

```diff
@@ -18,6 +18,10 @@ public class TupleSortedSetTest extends JedisCommandsTestBase {
   final byte[] be = { 0x0E };
   final byte[] bf = { 0x0F };
 
+  public TupleSortedSetTest() {
+    super(null);
```

Comment:
I'd rather not a have such a default, because that is more the exception case than the norm. I added comments stating the intention, and I made more tests parameterized, where possible.

Reply:
I mean, when new tests will be added, it should be obvious that the default is to parameterize them, so it should be an explicit action: either parameterize, either make them run a single version of the protocol, but don't risk of doing nothing because of a default constructor.